docs/wiki/99 - Developers/UnitTestGuide.md

```diff
@@ -0,0 +1,646 @@
+# **Unit Testing 101: How to Write Tests for LORIS**
+
+####  Alexandra Livadas, [Google Summer of Code](https://alexandralivadas.github.io/pages/gsoc-2019.html)
```

Comment:
Ok let's remove your author name from the top since it's now part of the markdown codebase.  we'll keep it below. 
```suggestion
```